pre_processing/read_ecmwf_wind_nc.F90

```diff
@@ -161,23 +161,23 @@ subroutine read_ecmwf_wind_nc_file(nwp_path, ecmwf)
             allocate(ecmwf%lat(ecmwf%ydim))
             call ncdf_read_array(fid, name, ecmwf%lat)
          end if
-      case('U10', 'U10M')
+      case('U10', 'U10M', 'u10')
```

Comment:
Can't we remove U10M since the HR reader is gone?

Reply:
Yes, I think so. Done!